mmv1/third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -2178,3 +2291,22 @@ var TEST_SIMPLE_CSV_SCHEMA = `[
       "type": "INT64"
     }
   ]`
+var TEST_INVALID_SCHEMA_NOT_JSON = `
+	not a valid table schema
+	`
+var TEST_INVALID_SCHEMA_NOT_JSON_LIST = `
```

Comment:
Quick question: If the schema is not a JSON_LIST, but a valid JSON, do we still return an error back?

Reply:
Sounds good, thanks for explaining. Looks like we can include this change in our major release we're preparing for.

Reply:
Thank you. I'll hold off the merge after getting an approval. I have another change waiting for the major release too. Will wait for more instructions in July.

Reply:
Following up again to see if guidance for the next major release has been published, and when to expect it otherwise.

Reply:
Thanks for the follow-up, I will re-trigger a test run when we are approaching the major release.